src/main/java/Formatter.java

```diff
@@ -0,0 +1,20 @@
+public class Formatter {
+    public String getPadej(double dividedSum) {
+        int lastTwoDig = (int) dividedSum % 100;
+        int lastDig = (int) dividedSum % 10;
+        if (lastTwoDig >= 11 && lastTwoDig <= 13) {
```

Comment:
```suggestion
        if (lastTwoDig >= 11 && lastTwoDig <= 20) {
```
Тут вот так скорее всего. Можно и до 14 включительно проверять, но число 14 должно попасть, иначе сейчас на 14 выдаёт "рубля", а нужно "рублей"